README.md

```diff
@@ -121,7 +121,7 @@ Keep in mind that for each new terminal session, you will need to **activate** t
 Now you can install the Python dependencies, to do so run:
 
 ```bash
-python3 -m pip install -U -r requirements.txt
+pip install -r requirements.txt
```

Comment:
I'd keep `-U` "upgrade already installed packages"
```suggestion
pip install -U -r requirements.txt
```

out of curiosity do you have a source for pip is always in venvs

Reply:
Copied from the other thread:

The official docs at https://docs.python.org/3/library/venv.html say

Changed in version 3.4: Installs pip by default, added the --without-pip and --copies options

and later it suggests using pip install -r requirements.txt (though oddly, only in a warning)

Reply:
Sure, we can keep `-U` although it's not necessary for an install. Ideally I'd like people to understand how to use pip and not blindly copy commands, and keeping the commands as simple as possible helps with that, but I suppose it's fine